src/components/ProjectDetailPage/ProjectDetailPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+import { Grid, Card, CardTitle, CardBody, FlexItem, Flex } from '@patternfly/react-core';
+import { PageLayout } from './../PageLayout/PageLayout';
+import { useDataContainer } from '../../containers/DataContainer/useDataContainer';
+import { projectService } from '../../services/projectService';
+import { DataContainer } from '../../containers/DataContainer/DataContainer';
+import { ActionHeader } from '../ActionButton/ActionButton';
+
+const ProjectAttribute = ({ attribute, value }: { attribute: string; value: string }) => {
```

Comment:
let's use shorter syntax in cases like this, `{ return` part can be skipped, see for example https://github.com/project-ncl/pnc-web-ui-react/blob/main/src/components/EmptyStates/EmptyStateCard.tsx#L13